ramls/schemas/quickExportResponse.json

```diff
@@ -0,0 +1,20 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "Quick export response",
+  "type": "object",
+  "additionalProperties": false,
+  "properties": {
+    "jobExecutionId": {
+      "description": "UUID of the job execution, that is used for quick export",
+      "type": "string"
```

Comment:
Not a big thing but why didn't we add the UUID like we usually do with pattern or infact we already have it in jobExecution.json?

Reply:
we already have it in jobExecution, but if you want I will add it in the scope of my current story